crates/agglayer-config/tests/validate_deserialize.rs

```diff
@@ -88,3 +89,59 @@ fn prover_grpc_max_decoding_message_size() {
 
     assert_eq!(config.grpc.max_decoding_message_size, 100 * 1024 * 1024);
 }
+
+#[test]
+fn network_prover() {
+    let input = "./tests/fixtures/valide_config/prover_config_network_prover.toml";
+    let config = ProverConfig::try_load(Path::new(input)).unwrap();
+
+    assert_eq!(
+        config.primary_prover,
+        AgglayerProverType::NetworkProver(NetworkProverConfig {
+            proving_request_timeout: Some(std::time::Duration::from_secs(300)),
+            proving_timeout: std::time::Duration::from_secs(600),
+        })
+    );
+
+    assert_eq!(config.fallback_prover, None);
+}
+
+#[test]
+fn cpu_prover() {
+    let input = "./tests/fixtures/valide_config/prover_config_cpu_prover.toml";
```

Comment:
```suggestion
    let input = "./tests/fixtures/validate_config/prover_config_cpu_prover.toml";
```

And similar for the whole file, won't copy further

Reply:
@Ekleog-Polygon Was already set by Simon before, didn't want to ruin the French touch :)